symphony/app/fbcnms-projects/magmalte/app/views/equipment/GatewayDetailStatus.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+import type { KPIRows } from '../../components/KPIGrid'
+import type { lte_gateway } from '@fbcnms/magma-api'
+
+import isGatewayHealthy from '../../components/GatewayUtils'
+import KPIGrid from '../../components/KPIGrid'
+import React from 'react'
```

Comment:
re-arrange this such that modules imports (uppercase) are in a block, and functions (lowercase) are in another
```suggestion
import KPIGrid from '../../components/KPIGrid'
import React from 'react'

import isGatewayHealthy from '../../components/GatewayUtils'
```